code/ui/manager/src/components/sidebar/Heading.tsx

```diff
@@ -44,13 +44,27 @@ const SkipToCanvasLink = styled(Button)(({ theme }) => ({
   '@media (min-width: 600px)': {
     display: 'block',
     position: 'absolute',
-    width: '100%',
-    padding: '10px 15px',
     fontSize: theme.typography.size.s1,
     zIndex: 3,
-    transform: 'translate(0,-100px)',
+    border: 0,
+    width: 1,
+    height: 1,
+    padding: 0,
+    margin: -1,
+    overflow: 'hidden',
+    clip: 'rect(0, 0, 0, 0)',
+    whiteSpace: 'nowrap',
+    wordWrap: 'normal',
```

Comment:
This style is Visually Hidden. It is an element while still allowing it to be exposed to assistive technologies (such as screen readers).

ref: https://getbootstrap.com/docs/5.0/helpers/visually-hidden/